creator-node/src/components/replicaSet/syncQueueComponentService.js

```diff
@@ -1,17 +1,9 @@
 /**
  * Enqueues sync operation into syncQueue for provided walletPublicKeys against provided creatorNodeEndpoint
  */
-const enqueueSync = async ({
-  serviceRegistry,
-  walletPublicKeys,
-  creatorNodeEndpoint,
-  forceResync
-}) => {
-  await serviceRegistry.syncQueue.enqueueSync({
-    walletPublicKeys,
-    creatorNodeEndpoint,
-    forceResync
-  })
+const enqueueSync = async (params) => {
```

Comment:
nit and nbd either way but this feels like a bit of a step backward to have the params not be explicit if there's no docstring or typescript - wdyt?

Reply:
so the reason why I left it as params is bc when I updated the params, I had to change it in two places. I figure for future extensibility/changes, it might be in our best interests to leave it as a `param` instead. and in the sync queue file, the params are explicit so at least there will be named params there. 

I can add a doc string if it makes it clearer!

Reply:
my 2c here i also think this is step backward - explicit param definition is much more readable, and don't think future changes is a reason to avoid, in fact the opposite future changes is a reason to keep explicit so dev has to be aware and change
esp below on lines 9-10 where you destructure params obj right after, seems better to just make explicit in arg?

Reply:
I dont think it's a step backwards here. again, if we were to update the params, we would need to do it in two places: into the wrapper function and into enqueueSync. as a wrapper function, I especially don't think we need this layer of granularity since the actual enqueueSync function itself does extract the params necessary and passes it to the job. I think this level of granularity will hurt us as its more surface area to update code if params change, like it did in this pr

Reply:
yeh its a fair point

below on `processSyncOfTypeImmediate` you immediately destructure the param so seems in that case would be better to just do it in the signature?

also per your above comment that you'll add a docString here, then any future changes would still require a change in 2 places, so seems moot?

Reply:
sure

Reply:
unresolving this thread since jsdoc still missing
also i thought conclusion here was to keep explicit params since jsdoc would require them anyway but its fine